sklearn/ensemble/tests/test_forest.py

```diff
@@ -73,6 +73,19 @@
     random_state=0,
 )
 
+# Imbalanced classification sample used for testing imbalanced criteria
+imbl_minority_class_ratio = 0.05
```

Comment:
It `imbl_minority_class_ratio` is to be used as a global constant, I think it conventionally has to be capitalized.

Reply:
done